mreg_cli/errorbuilder.py

```diff
@@ -107,12 +108,16 @@ def can_build(self) -> bool:  # noqa: D102
         return True
 
 
+DEFAULT_OFFSET = (-1, -1)
```

Comment:
Nitpicky, but is this really a "default_offset" or a "word_not_found_result"?

Reply:
You are right that it is more of a sentinel value - albeit one that still contains valid indices. I have refactored it to be more clear now.

Reply:
The refactored code looks good! Thanks!